contracts/staking/Staking.sol

```diff
@@ -518,17 +518,6 @@ contract Staking is StakingV2Storage, GraphUpgradeable, IStaking, Multicall {
         emit SlasherUpdate(msg.sender, _slasher, _allowed);
     }
 
-    /**
-     * @dev Set an address as allowed asset holder.
-     * @param _assetHolder Address of allowed source for state channel funds
-     * @param _allowed True if asset holder is allowed
-     */
-    function setAssetHolder(address _assetHolder, bool _allowed) external override onlyGovernor {
-        require(_assetHolder != address(0), "!assetHolder");
-        assetHolders[_assetHolder] = _allowed;
-        emit AssetHolderUpdate(msg.sender, _assetHolder, _allowed);
```

Comment:
We can also remove this event `event AssetHolderUpdate(address indexed caller, address indexed assetHolder, bool allowed);`

Reply:
good point! done